pkg/registry/csv.go

```diff
@@ -340,3 +340,50 @@ func (csv *ClusterServiceVersion) GetOperatorImages() (map[string]struct{}, erro
 
 	return images, nil
 }
+
+// setCSVField sets a top level field on the CSV spec
+func (csv *ClusterServiceVersion) setCSVSpecField(key string, newValue interface{}) (err error) {
+	var objmap map[string]*json.RawMessage
+	if err := json.Unmarshal(csv.Spec, &objmap); err != nil {
+		return err
```

Comment:
nit: shadowing the return parameter with a new declaration inside the function

seems like not having a named error return type makes this function more clear, if not using a bare return

Reply:
in this case, the shadowed variable is scoped only to the if statement, but I agree with the point on clarity